gateway/src/constructors/express/routes/api/v1/rpc/chainHandler.js

```diff
@@ -0,0 +1,55 @@
+
+const axios = require('axios')
+const router = require('express').Router()
+
+const { getRPCServiceAddress } = require('@/share/lib')
+
+
+/**
+ * @title chainHandler 
+ * @dev   Handles the ethreum,bitcoin and bsc rpc endpoints
+ * @dev   chainValue : identifies the chain 
+ * @dev   networkValue: identifies the network type
+ */
+
+const chainHandler = module.exports = ({ }) => {
+
+  router.post('/', async (req, res) => {
+
+
+    const baseUrl = req.baseUrl
+    const urlValue = baseUrl.split('/')[4]
+
+
+    try {
+      let chainValue, networkValue
+
+      if (urlValue === 'ethereum') {
+        chainValue = 'ethereum'
+        networkValue = 'rinkeby'
+      }
+      else if (urlValue === 'bitcoin') {
+        chainValue = 'bitcoin'
+        networkValue = 'testnet'
+      }
+      else if (urlValue === 'binance-smart-chain' || 'bsc') {
```

Comment:
`urlValue === 'binance-smart-chain' || urlValue === 'bsc'`

or `['binance-smart-chain', 'bsc].includes(urlValue)`

Reply:
not necessary if we use the above logic